package.json

```diff
@@ -13,7 +13,9 @@
     "airtable": "^0.10.1",
     "bottleneck": "^2.19.5",
     "dotenv": "^8.2.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "full-icu": "^1.3.1",
```

Comment:
Interesting.  What does this do?

Reply:
Allows [use of locales](https://moment.github.io/luxon/docs/manual/install.html#node). Didn't end up actually using it - originally pulled in to try to solve some timezone stuff, but would likely need it if get into language stuff so just left